test/integration/targets/vyos_interfaces/vars/main.yaml

```diff
@@ -0,0 +1,233 @@
+---
+merged:
+  before:
+    - name: "eth0"
```

Comment:
same, lets drop eth0

Reply:
This is not a test. This is the fixtures file.